config.yaml

```diff
@@ -83,4 +83,16 @@ core_config_version: 0
 
 
 # (OPTIONAL | Default: ""). Used to prepend a base path to all APIs when querying the core.
-# base_path:
\ No newline at end of file
+# base_path:
+
+# (OPTIONAL | Default: "ARGON2"). The password hashing algorithm to use. Values are "ARGON2" | "BCRYPT"
+# password_hashing_alg:
+
+# (OPTIONAL | Default: 3). Number of iterations for argon2 password hashing
```

Comment:
I think these defaults look fine, this is the recommendation for memory-constrained envs. (Check out the [RFC](https://datatracker.ietf.org/doc/rfc9106/), Section 7.4)
We could also test (at least manually) how many parallel login attempts we can handle on an example PC (or AWS instance).

Reply:
I think a periodically updated list about this and/or how much time each login takes with some recommended settings could be nice in the docs.
Also, check out: https://www.ory.sh/choose-recommended-argon2-parameters-password-hashing/

Reply:
> I think a periodically updated list about this and/or how much time each login takes with some recommended settings could be nice in the docs.

Agreed.

Reply:
> I think a periodically updated list about this and/or how much time each login takes with some recommended settings could be nice in the docs.

When you say "periodically updated list", what do you mean?

Reply:
I have changed the default configs. Please see once again